infrastructure/sentry-subgraph/schema/events/esXai.graphql

```diff
@@ -0,0 +1,14 @@
+
+type RedemptionRequest @entity(immutable: false) {
+    id: String!
+    sentryWallet: SentryWallet! @index # User address, must match the address field in SentryWallet
```

Comment:
Does this need to be required ?

Reply:
I don't understand the question? Does what need to be required?
The Sentry Wallet?
The index param?

The Sentry wallet is required to connect the redemption request to the user.
The index is not required but will improve queries for specific addresess.

Reply:
Im talking about the field `sentryWallet` being marked as required / not null with the `"!"`

Reply:
Wouldn't each redemption need to have a Sentry Wallet associated with it?
Why would it not be required?

Reply:
Makes sense, I did not realize we will create a sentry Wallet for every user that does redemptions. This should be fine.